bundle/edu.gemini.itc.shared/src/main/java/edu/gemini/itc/shared/SourceDefinition.java

```diff
@@ -117,15 +120,15 @@ public double getBBTemp() {
         return ((BlackBody) distribution).temperature();
     }
 
-    public Wavelength getELineWavelength() {
+    public Length getELineWavelength() {
         return ((EmissionLine) distribution).wavelength();
     }
```

Comment:
As @swalker2m notes, this is kind of odd; we might at least want to say `type Wavelength = Length` somewhere.